oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -109,6 +109,40 @@ public class ComputeEngineCredentials extends GoogleCredentials
   static final int MAX_COMPUTE_PING_TRIES = 3;
   static final int COMPUTE_PING_CONNECTION_TIMEOUT_MS = 500;
 
+  public enum AuthTransport {
+    // Authenticating to Google APIs via DirectPath
+    ALTS("alts"),
+    // Authenticating to Google APIs via GFE
+    MTLS("mtls");
+
+    private final String label;
+
+    private AuthTransport(String label) {
+      this.label = label;
+    }
+
+    public String getLabel() {
+      return label;
+    }
+  }
+
+  public enum BindingEnforcement {
+    // Binding enforcement will always happen, irrespective of the IAM policy.
+    ON("on"),
+    // Binding enforcement will depend on IAM policy.
+    IAMPOLICY("iam-policy");
```

Comment:
```suggestion
    IAM_POLICY("iam-policy");
```

I believe the convention is underscore to separate the words for constants

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1558/commits/f36a19cd4e94898e896950304296160e2a88d685